README.md

```diff
@@ -117,14 +119,16 @@ The following environment variables are supported:
 You can configure the environment variables in the workflow file like this:
 
 ```yaml
-        env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-          LABELS: "!wip,!work in progress,documentation-updated"
-          AUTOMERGE: "ready-to-merge"
-          AUTOREBASE: "ready-to-rebase-and-merge"
-          MERGE_METHOD: "squash"
-          MERGE_FORKS: "false"
-          COMMIT_MESSAGE_TEMPLATE: "pull-request-description"
+env:
+  GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+  LABELS: "!wip,!work in progress,documentation-updated"
+  MERGE_LABEL: "ready-to-merge"
+  UPDATE_LABEL: "ready-to-update"
+  UPDATE_AND_MERGE_LABEL: "ready-to-update-and-merge"
+  MERGE_METHOD: "squash"
+  UPDATE_METHOD: "rebase"
+  MERGE_FORKS: "false"
+  COMMIT_MESSAGE_TEMPLATE: "pull-request-description"
```

Comment:
Indent should be kept, as it's easier to copy & paste then, IMO!

Reply:
Ah sorry; was my IDE auto formatting